docker-compose.yaml

```
@@ -0,0 +1,80 @@
+# Nautilus Wallet demo and staging deployment
+
+networks:
+  vault-dev:
+  vault-staging:
+  vault-main:
+
+services:
+  # Docs: https://github.com/nginx-proxy/nginx-proxy
+  nginx-proxy:
+    image: nginxproxy/nginx-proxy:alpine
+    networks:
+      - vault-dev
+      - vault-staging
+      - vault-main
+    ports:
+      - "8080:80"
+      - "8443:443"
```

Why not just the following
```suggestion
      - "80:80"
      - "443:443"
```

---

"[ci: change ports due to clash with backend-services](https://github.com/ntls-io/nautilus-trusted-compute-deployment/pull/1/commits/f22d63bd12ff1c2fd5ff53d93814cca77f901d6d)"

---

I strongly recommend that the backend-services be included in this configuration.  Otherwise there is no record of it and we'll be using a non-standard port for our deployments.

---

I've done the recommendation in https://github.com/ntls-io/nautilus-trusted-compute-deployment/pull/2